python/ray/experimental/state.py

```diff
@@ -410,12 +440,12 @@ def micros(ts):
       return int(1e6 * (ts - start_time))
 
     full_trace = []
-
     for task_id, info in task_info.items():
-      parent_info = task_info.get(tasks[task_id]["TaskSpec"]["ParentTaskID"])
+      taskid = ray.local_scheduler.ObjectID(hex_to_binary(task_id))
```

Comment:
Words in variable names need to be separated with underscores. This case is a bit unfortunate, but maybe `task_id`, and the first instance can be changed to `task_id_hex`.